src/Saritasa.NetForge.Mvvm/ViewModels/EditEntity/EditEntityViewModel.cs

```diff
@@ -80,7 +80,7 @@ public override async Task LoadAsync(CancellationToken cancellationToken)
                         IsValueGeneratedOnAdd: false,
                         IsValueGeneratedOnUpdate: false,
                         IsHiddenFromListView: false,
```

Comment:
I found out that this problem not only for navigations, we can't hide any property using `SetIsHidden` and `SetIsHiddenFromDetails` methods. So, try this code and ensure it works for any property.
```suggestion
                        IsHidden: false,
                        IsHiddenFromDetails: false
```